### PR TITLE
Add an optional section to ask reproducible codes

### PR DESCRIPTION
**Pull request by nzw0301**
## Motivation

Optuna users ask Optuna's usage with other libraries, but sometimes we find a difficult to answer the question since we cannot see a context such as codes. 

## Description of the changes

Add an optional section to provide reproducible codes by following [Optuna's bug-report issue template](https://github.com/optuna/optuna/blob/master/.github/ISSUE_TEMPLATE/bug-report.md).

### CODE REVIEW

**Comment by HideakiImamura** [comment]
@himkt Could you review this PR if you have time?

**Review by HideakiImamura — APPROVED** [review]
LGTM!

File: .github/ISSUE_TEMPLATE/questions-help-support.md [hunk 1]
```diff
@@ -9,6 +9,35 @@ assignees: ''
 
 <!-- Please write your question here. -->
 
+## Information for reproducibility (optional)
+
+<!-- If you ask how to use Optuna's functionality in your concrete codes, the minimal reproducible code and related info are really helpful to answer your question.  -->
+
+### Environment
+
+- Optuna version:
```
Comment (himkt) [inline .github/ISSUE_TEMPLATE/questions-help-support.md:18]: [nits]
I'm wondering if putting a sample script like below would be helpful to collect consistent information.

```python
import platform

import optuna


print(f"OS: {platform.platform()}")
print(f"Python: {platform.python_version()}")
print(f"Optuna: {optuna.__version__}")
```

**Review by himkt — APPROVED** [review]
LGTM!

**Comment by himkt** [comment]
Let me merge. As a follow-up, I'll create a thin module to show runtime information.
Thank you for improving the issue template!

**Pull request merged**